src/auto/pixbuf.rs

```diff
@@ -103,13 +103,6 @@ impl Pixbuf {
         }
     }
 
-    #[cfg(any(feature = "v2_36_8", feature = "dox"))]
-    pub fn calculate_rowstride(colorspace: Colorspace, has_alpha: bool, bits_per_sample: i32, width: i32, height: i32) -> i32 {
```

Comment:
This function should not really disappear. It's useful and definitely still available 2.36.11 (the latest release) for example.